src/reducers/SupplierCreditReducer.js

```diff
@@ -68,9 +68,11 @@ export const SupplierCreditReducer = (state = initialState(), action) => {
 
       const item = UIDatabase.get('Item', itemId);
 
+      const query = 'numberOfPacks > 0 && supplier != null && supplier.isVisible == true';
+
       return {
         ...state,
-        batches: mapBatchToObject(item?.batches?.filtered('numberOfPacks > 0') ?? []),
+        batches: mapBatchToObject(item?.batches?.filtered(query)),
```

Comment:
Think where possible it would be good to use more "self-documenting" identifiers for query string variables? E,g, `queryNonEmptyVisibleBatches`?

Reply:
went with this I don't like either 🤣 
108b0a6